### PR TITLE
VD-783: Prepare django-triggers for Django 2.0

### PR DESCRIPTION
**Pull request by pieterdd**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -8,6 +8,10 @@ env:
 python:
    - "2.7"
    - "3.6"
+matrix:
+  exclude:
+  - python: "2.7"
+    env: DJANGO='>=2.0,<2.1'
```
Comment (joncasdam) [inline .travis.yml]: same here, `DJANGO='>=2.0'` wouldn't be enough?

File: requirements/requirements.txt [hunk 2]
```diff
@@ -1,4 +1,4 @@
-Django>=1.8,<1.12
+Django>=1.8,<2.1
```
Comment (joncasdam) [inline requirements/requirements.txt]: why the limit for `2.1`?

File: .travis.yml [hunk 3]
```diff
@@ -4,13 +4,18 @@ env:
    - DJANGO='>=1.9,<1.10'
    - DJANGO='>=1.10,<1.11'
    - DJANGO='>=1.11,<1.12'
+   - DJANGO='>=2.0,<2.1'
```
Comment (joncasdam) [inline .travis.yml]: DJANGO='>=2.0' wouldn't be enough?

**Comment by pieterdd** [comment]
Talked irl.

**Pull request merged**